regolith/builders/gradebuilder.py

```diff
@@ -43,22 +43,23 @@ def render(self, tname, fname, **kwargs):
         ctx["root"] = ctx.get("root", os.path.relpath("/", os.path.dirname(fname)))
         try:
             result = template.render(ctx)
-        except:
+        except Exception:
             type, value, tb = sys.exc_info()
             traceback.print_exc()
             pdb.post_mortem(tb)
         with open(os.path.join(self.bldir, fname), "wt", encoding="utf-8") as f:
             f.write(result)
 
     def latex(self):
-        rc = self.rc
         for course in self.gtx["courses"]:
             if not course.get("active", True):
                 continue
             course_id = course["_id"]
             stats = self.makestats(course)
             asgn = filter((lambda x: course_id in x["courses"]), self.gtx["assignments"])
-            catfunc = lambda x: x["category"]
+
+            def catfunc(x):
+                return x["category"]
             asgn = sorted(asgn, key=catfunc)
```

Comment:
I think this would be cleaner if you just move the lambda to this line rather than define it as a function that is not used elsewhere.  So
```
asgn = sorted(asgn, key=lambda x: x["category"])
```
and delete line 61.  Something like that should work, is cleaner and should pass ci.